### ISSUE TITLE
Missing tests?

### ISSUE THREAD

**Issue by deniederhut** [ORNL/tx2#4]
I'm not seeing where the test cases are for this repo. Am I looking in the wrong place?

**Comment by deniederhut** [comment]
Yeah that's a great question. I think we'll want to kick this up the chain.

**Comment by fabian-s** [comment]
It's sub-par but acceptable, I'd say. The examples provide a manual way to check that the tool is working as expected. Obviously proper unit or regression test and adding CI to the repo would be preferable

@deniederhut would you say that these two notebooks cover TX21's central functionality?
If so, I'd recommend to cross that item off the checklist. 
If not, please identify what's missing, and then we can unblock once @WildfireXIII has added examples for those parts.

**Comment by deniederhut** [comment]
Thanks for chiming in! I've tried that first notebook and it's failing with an error. I've filed a separate ticket about it, so I think we can close this one.

**Comment by WildfireXIII** [comment]
Updating this thread for awareness - we ended up adding some basic unit tests to aid in some of our updates, the test suite can be run via `pytest` from the root directory. (I've updated the README to include this as well.)
